packages/block-library/src/video/edit.native.js

```diff
@@ -377,7 +377,6 @@ export default compose( [
 
 		const lastBlockInserted = getLastBlockInserted();
 		const wasBlockJustInserted =
-			!! lastBlockInserted &&
 			lastBlockInserted.clientId === clientId &&
 			lastBlockInserted.source === 'inserter_menu';
```

Comment:
@fluiddot do we have to duplicate 
```
lastBlockInserted.clientId === clientId &&
lastBlockInserted.source === 'inserter_menu';
```
across all three components? Could it be placed within the [selector](https://github.com/WordPress/gutenberg/blob/1164a3338fa9347a52bffcc34283d0986daf2e2c/packages/block-editor/src/store/selectors.js#L2114) where clientId and source get passed in as params ? Let me know.

Reply:
Yeah, that's a good idea, we could use the selector you created [`wasBlockJustInserted`](https://github.com/WordPress/gutenberg/blob/1164a3338fa9347a52bffcc34283d0986daf2e2c/packages/block-editor/src/store/selectors.js#L2097-L2106).

Reply:
I've just updated the `wasBlockJustInserted` selector in this [commit](https://github.com/WordPress/gutenberg/pull/30238/commits/f08846a4d511354051859d74a56fc0fac89d92a8).